app/src/main/java/com/github/sdp_begreen/begreen/models/Post.kt

```diff
@@ -0,0 +1,10 @@
+package com.github.sdp_begreen.begreen.models
+
+import android.graphics.Bitmap
+
+class Post(private val title: String, private val image: Bitmap) {
```

Comment:
Looks like you could omit `private` here and omit the explicit `get...` functions, since `val` is immutable?